example/testapp/models.py

```diff
@@ -9,10 +9,13 @@ class Car(models.Model):
     def __unicode__(self):
         return self.plate
 
+class m2mprint:
+    def __unicode__(self):
```

Comment:
`__unicode__` method is being used almost everywhere in this package. Shall I replace all the occurrences?